packages/cli/src/constructs/phone-call-alert-channel.ts

```diff
@@ -0,0 +1,43 @@
+import { AlertChannel, AlertChannelProps } from './alert-channel'
+import { Session } from './project'
+
+export interface PhoneCallAlertChannelProps extends AlertChannelProps {
+  /**
+   * The phone number where to send the alert notifications.
```

Comment:
@maxigimenez we should put a link here to our docs which gives more info about which countries are supported

Reply:
@umutuzgur Good call. Done